generator/standalone.cc

```diff
@@ -85,26 +85,26 @@ bool ConvertCommandLineArgs(int argc, char const* const argv[],
     std::vector<std::string> arg =
         absl::StrSplit(argv[i], absl::MaxSplits('=', 1));
     if (arg.size() <= 1) {
-      args.emplace_back(argv[i]);
```

Comment:
Nit: please add a null check on args at the top of the function now that it's a pointer

Reply:
Skipping the check (as we discussed offline).